internal/repository/prune.go

```diff
@@ -634,7 +634,8 @@ func deleteFiles(ctx context.Context, ignoreError bool, repo restic.RemoverUnpac
 				return err
 			}
 		}
+		bar.Add(1)
 		printer.VV("removed %v/%v\n", fileType, id)
```

Comment:
That log output is also wrong if the file deletion failed, but `ignoreError` is set.